src/ArethusaConfig.js

```diff
@@ -34,7 +34,7 @@ const defaultConfig = {
   navbar: false,
   resources: {
     Gardener: {
-      route: `${process.env.PUBLIC_URL}/xml/:doc`,
+      route: `./:doc`,
```

Comment:
I was able to change this without breaking anything. But if I get rid of it, the widget stops working. I can dig into the Arethusa source, but off the top of your head, do you know what `route` and `params` are used for?

Reply:
They are used by Arethusa resource factory (https://github.com/alpheios-project/arethusa/blob/widget/app/js/arethusa.core/factories/resource.js), which wraps an Angular 1 $resource (https://docs.angularjs.org/api/ngResource/service/$resource) object

As this resource is used by the Treebank Retriever, the route specifies the url template for a  treebank file and the params indicate what gets replaced in the template. 

I think whether or not it would work without a full path depends upon the underlying environment in which it gets deployed, i.e how the XHR request is interpreted